tests/console/validate_lvm_encrypt.pm

```diff
@@ -17,6 +17,11 @@ use testapi;
 use utils;
 use y2logsstep;
 use Test::Assert ':all';
+use version_utils qw(is_aarch64 is_storage_ng);
+
+sub is_not_storage_ng_is_aarch64 {
```

Comment:
minor thing, I think it doesn't help in this case to create a separate function unless you have a better naming, it doesn't follow 'verb + action' (I haven't seen before a double "is" in a function name) and doesn't save space if you include those conditions below.